jdaviz/core/helpers.py

```diff
@@ -273,96 +274,94 @@ def get_model_parameters(self, models=None, model_label=None, x=None, y=None):
 
         return parameters_cube
 
-    def show_inline(self):
-        """
-        Display the Jdaviz application inline in a notebook.  Note this is
-        functionally equivalent to displaying the cell ``self.app`` in the
-        notebook.
-
-        See Also
-        --------
-        show_in_sidecar
-        show_in_new_tab
-        """
-        display(self.app)
-
-    def show_in_sidecar(self, **kwargs):
-        """
-        Display the Jdaviz application in a "sidecar", which by default is a tab
-        on the right side of the JupyterLab  interface.
-
-        Additional keywords not listed here are passed into the
-        ``sidecar.Sidecar`` constructor. See
-        `jupyterlab-sidecar <https://github.com/jupyter-widgets/jupyterlab-sidecar>`_
-        for the most up-to-date options.
+    def show(self, loc="inline", title=None):
+        """ Display the Jdaviz application.
 
         Parameters
         ----------
+        loc : str
+            The display location determines where to present the viz app.
+            Supported locations:
+
+            "inline": Display the Jdaviz application inline in a notebook.
+            Note this is functionally equivalent to displaying the cell
+            ``viz.app`` in the notebook.
+
+            "sidecar": Display the Jdaviz application in a separate JupyterLab window from the
+            notebook, the location of which is decided by the 'anchor.' right is the default
+
+                Other anchors:
+
+                * ``sidecar:right`` (The default, opens a tab to the right of display)
+                * ``sidecar:tab-before`` (Full-width tab before the current notebook)
+                * ``sidecar:tab-after`` (Full-width tab after the current notebook)
+                * ``sidecar:split-right`` (Split-tab in the same window right of the notebook)
+                * ``sidecar:split-left`` (Split-tab in the same window left of the notebook)
+                * ``sidecar:split-top`` (Split-tab in the same window above the notebook)
+                * ``sidecar:split-bottom`` (Split-tab in the same window below the notebook)
+
+                See `jupyterlab-sidecar <https://github.com/jupyter-widgets/jupyterlab-sidecar>`_
+                for the most up-to-date options.
+
         title : str, optional
             The title of the sidecar tab.  Defaults to the name of the
             application; e.g., "specviz".
-        anchor : str
-            Where the tab should appear, by default on the right. Options are:
-            {sidecar_anchor_values}.
+
+            NOTE: Only applicable to a "sidecar" display.
 
         Returns
         -------
-        sidecar
-            The ``sidecar.Sidecar`` object used to create the tab.
+        scar : sidecar or `None`
```

Comment:
So you want to keep returning this?

Reply:
Consensus earlier was to not. Working on that now!

Reply:
Implemented in https://github.com/spacetelescope/jdaviz/pull/965/commits/117d2ab00181f7ac4244560fe9a9927672c7b930